wtpython/display.py

```diff
@@ -169,7 +167,7 @@ async def on_startup(self, event: events.Startup) -> None:
         footer.add_key("d", "Open in Browser")
         footer.add_key("f", "Search Google")
 
-        await self.set_focus(self.body._page)
+        self.body.focus()
```

Comment:
This doesn't seem to be focusing the ScrollView for me on WSL. I need to click into it to get it to scroll - changing `._page` to `.page` works though i.e.
```suggestion
        await self.set_focus(self.body.page)
```

Reply:
Sorry about that, I didn't realize the purpose of those lines.